api/client/cli.go

```diff
@@ -105,15 +105,21 @@ func NewDockerCli(in io.ReadCloser, out, err io.Writer, clientFlags *cli.ClientF
 
 		switch len(hosts) {
 		case 0:
-			hosts = []string{os.Getenv("DOCKER_HOST")}
+			defaultHost := os.Getenv("DOCKER_HOST")
```

Comment:
Isn't this `defaultHost` throw away? I believe it'll be scoped to just this block, but if not the assignment on like 116 will change it.  I other words, I think the original code (the one liner) will have the same net result.  no?